documentation/src/main/asciidoc/topics/xml/persistence_sql_query.xml

```diff
@@ -3,20 +3,16 @@
     <query-jdbc-store xmlns="urn:infinispan:config:store:jdbc:{schemaversion}"
                       dialect="POSTGRES"
                       shared="true">
-      <connection-pool connection-url="jdbc:postgresql://localhost:5432/postgres"
-                       username="postgres"
-                       password="changeme"
-                       driver="org.postgresql.Driver"/>
       <queries key-columns="isbn">
-        <select-single>SELECT isbn, title FROM books WHERE isbn = :isbn</select-single>
-        <select-all>SELECT isbn, title FROM books</select-all>
-        <delete>DELETE FROM books WHERE isbn = :key</delete>
+        <select-single>SELECT books.id, books.isbn, title, authors.name, authors.country FROM books JOIN authors ON books.author_id = authors.id WHERE books.id=:id</select-single>
```

Comment:
```suggestion
        <select-single>SELECT books.id, books.isbn, books.title, authors.name, authors.country FROM books JOIN authors ON books.author_id = authors.id WHERE books.id=:id</select-single>
```

Reply:
This whole section here has the same issues as the others I listed with the keys not matching and not needing JOIN on the size command as well.